### DISCUSSION TITLE
How to make a custom design combobox using the .slint script?

### DISCUSSION THREAD

**Discussion by kiptunor** [slint-ui/slint#3863] [Q&A]
I'm working on my own custom style of widgets and I'm not sure how to make a custom style.

**Comment by ogoffart** [comment]
We have an example of this in the printer demo:

https://github.com/slint-ui/slint/blob/e248093236e6568506545ea1470a4c57d1ba3115/examples/printerdemo/ui/common.slint#L170

**Reply by kiptunor** [reply]
Thank you!
